$OpenBSD$

OpenBSD paths: /etc/opt/simplex -> /etc/simplex/smp-server,
/var/opt/simplex -> /var/simplex/smp-server.

--- apps/smp-server/Main.hs
+++ apps/smp-server/Main.hs
@@ -7,10 +7,10 @@
 import SMPWeb (smpGenerateSite)
 
 defaultCfgPath :: FilePath
-defaultCfgPath = "/etc/opt/simplex"
+defaultCfgPath = "/etc/simplex/smp-server"
 
 defaultLogPath :: FilePath
-defaultLogPath = "/var/opt/simplex"
+defaultLogPath = "/var/simplex/smp-server"
 
 logCfg :: LogConfig
 logCfg = LogConfig {lc_file = Nothing, lc_stderr = True}
